api/layer/cors.go

```diff
@@ -0,0 +1,40 @@
+package layer
+
+import (
+	"context"
+
+	"github.com/nspcc-dev/neofs-s3-gw/api/data"
+	"github.com/nspcc-dev/neofs-s3-gw/api/errors"
+)
+
+func (n *layer) PutBucketCORS(ctx context.Context, p *PutCORSParams) error {
+	s := &PutSystemObjectParams{
+		bktInfo:  p.BktInfo,
+		objName:  p.BktInfo.CORSObjectName(),
+		metadata: map[string]string{},
+		prefix:   "",
+		payload:  p.CORSConfiguration,
+	}
+
+	if _, err := n.putSystemObject(ctx, s); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (n *layer) GetBucketCORS(ctx context.Context, bktInfo *data.BucketInfo) ([]byte, error) {
+	obj, err := n.getSystemObject(ctx, bktInfo, bktInfo.CORSObjectName(), true)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We can check err and if it is `errors.ErrNoSuchkey` then return `errors.ErrNoSuchCORSConfiguration`